specs/ChainSpec/LightClient.md

```diff
@@ -0,0 +1,86 @@
+# Light Client
+
+The state of the light client is defined by:
+
+1. `BlockHeaderInnerLiteView` for the current head (which contains `height`, `epoch_id`, `next_epoch_id`, `prev_state_root`, `outcome_root`, `timestamp` and the hash of the block producers set for the next epoch `next_bp_hash`);
+2. The set of block producers for the current and next epochs.
+
+Light clients operate by periodically fetching instances of `LightClientBlockView` via particular RPC end-point desdribed [below](#rpc-end-point).
+
+```rust
+pub struct ValidatorStakeView {
+    pub account_id: AccountId,
+    pub public_key: PublicKey,
+    pub stake: Balance,
+}
+
+pub struct LightClientBlockView {
+    pub prev_block_hash: CryptoHash,
+    pub next_block_inner_hash: CryptoHash,
+    pub inner_lite: BlockHeaderInnerLiteView,
+    pub inner_rest_hash: CryptoHash,
+    pub next_bps: Option<Vec<ValidatorStakeView>>,
```

Comment:
Because `next_bps` here is optional. Next BPs can be ~half of the size of the block, and is the same throughout the epoch, so if a light client queries blocks more frequently than once per epoch, there's no need to send the full set each time.